best-html-reporter.js

```diff
@@ -17,6 +17,7 @@ exports.init = function(config) {
     currentSuite;
   
   config.screenshots = config.screenshots || 'none';
+  config.reportDir = config.reportDir || __dirname + '/reports'
```

Comment:
what if the __dirname + '/reports' directory is not present? will it still work?

Reply:
done Raj.. :)